lib/picker.js

```diff
@@ -607,7 +607,21 @@ function PickerConstructor( ELEMENT, NAME, COMPONENT, OPTIONS ) {
         ) //endreturn
     } //createWrappedComponent
 
-
+    // taken from https://davidwalsh.name/javascript-debounce-function
+    function debounce(func, wait, immediate) {
```

Comment:
This should probably be defined outside of the PickerConstructor..maybe next to where we have `getRealEventTarget`

Reply:
Ok, will change this later today.